nemo/collections/common/tokenizers/japanese_tokenizers.py

```diff
@@ -45,14 +45,15 @@ def __init__(self, sp_tokenizer_model):
     def sp_tokenize(self, text: str) -> str:
         return ' '.join(self.sp_tokenizer.text_to_tokens(text))
     
-    def tokenize(self, text):
+    def tokenize(self, text, escape=False, return_str=False):
         """
         Detokenizes a list of sentencepiece tokens in Japanese
```

Comment:
Nit:  The comment should be "Tokenizes text through moses then sentencepiece" or possibly removed entirely.

Reply:
Fixed. Thanks :)